tests/helpers.py

```diff
@@ -127,6 +130,22 @@ def send_create_update_tx(name, ddo, flags, account):
     return txn_receipt, dt_contract, erc20_address
 
 
+def send_set_metadata_state_tx(ddo, account, state):
+    datatoken_address = ddo["dataToken"]
+
+    web3 = get_web3()
```

Comment:
you are using get_web3() a few times in this function, but you can use web3 directly